src/mpol/gridding.py

```diff
@@ -246,15 +246,17 @@ def _sum_cell_values_channel(self, uu, vv, values=None):
             A 2D array of size ``(npix, npix)`` in ground format containing the summed cell quantities.
         """
 
-        result = np.histogram2d(
+        bin_range = self.coords.vis_ext[:2]
```

Comment:
Could we make this code clearer to understand? For example, using `coords.v_bin_min`, and `coords.v_bin_max` and `coords.u_bin_min` etc..

Reply:
Sure thing! Somehow it didn't occur to me that those attributes were available. I'll push the changes shortly.